tests/adapters/switches/brocade_test.py

```diff
@@ -2146,38 +2166,61 @@ def test_remove_non_existent_dhcp_relay_server_fails(self):
 
 
 def vlan_display(vlan_id, name="[None]"):
-    return [
-        "PORT-VLAN {}, Name {}, Priority Level -, Priority Force 0, Creation Type STATIC".format(vlan_id, name),
-        "Topo HW idx    : 81    Topo SW idx: 257    Topo next vlan: 0",
-        "L2 protocols   : STP",
-        "Associated Virtual Interface Id: NONE",
-        "----------------------------------------------------------",
-        "No ports associated with VLAN",
-        "Arp Inspection: 0",
-        "DHCP Snooping: 0",
-        "IPv4 Multicast Snooping: Disabled",
-        "IPv6 Multicast Snooping: Disabled",
-        "No Virtual Interfaces configured for this vlan"
-    ]
+    return VlanDisplayBuilder(vlan_id, name).to_strings()
 
 
 def vlan_with_vif_display(vlan_id, vif_id, name="[None]"):
-    return [
-        "PORT-VLAN {}, Name {}, Priority Level -, Priority Force 0, Creation Type STATIC".format(vlan_id, name),
-        "Topo HW idx    : 81    Topo SW idx: 257    Topo next vlan: 0",
-        "L2 protocols   : STP",
-        "Associated Virtual Interface Id: {}".format(vif_id),
-        "----------------------------------------------------------",
-        "No ports associated with VLAN",
-        "Arp Inspection: 0",
-        "DHCP Snooping: 0",
-        "IPv4 Multicast Snooping: Disabled",
-        "IPv6 Multicast Snooping: Disabled",
-        "Ve{} is down, line protocol is down".format(vif_id),
-        "  Type is Vlan (Vlan Id: {})".format(vlan_id),
-        "  Hardware is Virtual Ethernet, address is 748e.f8a7.1b01 (bia 748e.f8a7.1b01)",
-        "  No port name",
-        "  Vlan id: {}".format(vlan_id),
-        "  Internet address is 0.0.0.0/0, IP MTU 1500 bytes, encapsulation ethernet",
-        "  Configured BW 0 kbps",
-    ]
+    return VlanDisplayBuilder(vlan_id, name).with_vif_display(vif_id).to_strings()
+
+class VlanDisplayBuilder:
```

Comment:
We haven'T been using the builder design in python so far because all this can be done with kwargs.

if instead of having a class you had 
```
def vlan_display(vlan_id=9, vlan_name="", tagged_ports=None, untagged_ports=None, with_vif=False)
```
you can achieve the same thing and it feels more python...
especially since you're not moving partial builders around